ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/LabelSpikes.java

```diff
@@ -32,33 +32,45 @@
  * Transform that maintains state per key in order to label suspicious clicks.
  */
 @SuppressWarnings("checkstyle:lineLength")
-public class LabelClickSpikes extends
+public class LabelSpikes extends
     PTransform<PCollection<KV<String, SponsoredInteraction>>, PCollection<KV<String, SponsoredInteraction>>> {
 
-  private final Integer maxClicks;
+  private final Integer maxInteractions;
   private final Long windowMillis;
-  private final Counter ghostClickCounter = Metrics.counter(LabelClickSpikes.class, "ghost_click");
+  private final Counter ghostClickCounter = Metrics.counter(LabelSpikes.class, "ghost_click");
```

Comment:
Lets rename this counter `ghost_<event_type>` so we can see the counter across the different event types